src/ChebyshevApproximation.cc

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory.
+// Written by J.-L. Fattebert, D. Osei-Kuffuor and I.S. Dunn.
+// LLNL-CODE-743438
+// All rights reserved.
+// This file is part of MGmol. For details, see https://github.com/llnl/mgmol.
+// Please also read this link https://github.com/llnl/mgmol/LICENSE
```

Comment:
could you make the header consistent with the other ones? They were changed recently

Reply:
Sure. I just pushed the correction to update the PR.